test/e2e/tests/account/lockdown.spec.js

```diff
@@ -77,7 +77,25 @@ describe('lockdown', function () {
           true,
           'The UI environment should be locked down.',
         );
+      },
+    );
+  });
 
+  it('the background environment is locked down (no-MV3)', async function () {
+    await withFixtures(
+      {
+        // The fixtures used here is arbitrary. Any fixture would do.
+        fixtures: new FixtureBuilder().build(),
+        ganacheOptions,
+        title: this.test.fullTitle(),
+      },
+      async ({ driver }) => {
+        if (
+          process.env.ENABLE_MV3 === 'true' ||
```

Comment:
Seems important that we still test lockdown in MV3? 🤔

Reply:
Agreed, but I don't think the method used by this test works for a service worker. This test opens the background HTML page, but there is no such page to open a service worker process. We'd need to find some other way to inject code into it from Selenium, or a new approach altogether.

In the meantime this is still a step forward, as this is completely broken/nonsensical for MV3 builds right now. Thanks for leaving that TODO comment about this @seaona !